src/support.js

```diff
@@ -440,16 +440,20 @@ function loadWebAssemblyModule(binary, flags) {
       if (legalized) {
         sym = 'orig$' + sym;
       }
+
+      var resolved = Module["asm"][sym];
+      if (!resolved) {
 #if WASM_BACKEND
-      sym = '_' + sym;
+        sym = '_' + sym;
 #endif
-      var resolved = Module[sym];
-      if (!resolved) {
-        resolved = moduleLocal[sym];
-      }
+        var resolved = Module[sym];
```

Comment:
Shouldn't this only be declared `var` once?

Reply:
Thanks, corrected it.